desktop/main.js

```diff
@@ -50,7 +60,7 @@ contextMenu({
         }),
         new MenuItem({
             label: Localize.translate(CONST.LOCALES.DEFAULT, 'desktopApplicationMenu.pasteAsPlainText'),
-            visible: parameters.isEditable && parameters.editFlags.canPaste,
+            visible: parameters.isEditable && parameters.editFlags.canPaste && clipboardHasText(),
```

Comment:
I think if this is used only once then we don't need a function. We can just `clipboard.readText().length > 0` here. Quick question, will `readText()` always return a string?

Reply:
[Yes](https://www.electronjs.org/docs/latest/api/clipboard#clipboardreadtexttype), it always returns a string